Helpers/RefreshTokenHelper.cs

```diff
@@ -0,0 +1,27 @@
+﻿using BookApp.API.Entities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Threading.Tasks;
+
+namespace BookAppStoreAPI.Helpers
+{
+    public class RefreshTokenHelper
+    {
+        public RefreshToken Generate_Refresh_Token()
+        {
+            RefreshToken refToken = new RefreshToken();
```

Comment:
Where is Refresh Token sent as the return type here? Also, where is the RefreshToken model?

Reply:
RefreshToken.cs file is entity class created right inside Entities folder. using object of refreshtoken.cs I am storing it into refresh token table.